modules/olm-updating-index-image.adoc

```diff
@@ -3,12 +3,30 @@
 // * operators/admin/olm-managing-custom-catalogs.adoc
 // * operators/admin/olm-restricted-network.adoc
 
+ifdef::openshift-origin[]
+:index-image-pullspec: quay.io/operator-framework/upstream-community-operators:latest
+:index-image: upstream-community-operators
+:tag: latest
+:catalog-name: upstream-community-operators
+:example-registry: example.com
+endif::[]
+ifndef::openshift-origin[]
+:index-image-pullspec: registry.redhat.io/redhat/redhat-operator-index:v4.6
+:index-image: redhat-operator-index
+:tag: v4.6
+:catalog-name: redhat-operators
+:example-registry: registry.redhat.io
+endif::[]
```

Comment:
Should these attributes be unset at the end of the file? E.g.,
> :!index-image-pullspec:
:!index-image:
:!tag:
:!catalog-name:
:!example-registry:

Reply:
Yes! Meant to, thanks.